src/methods.cpp

```diff
@@ -354,6 +355,10 @@ bool PyNeoDeviceEx_GetHandle(PyObject* object, void** handle)
         return false;
     }
     *handle = NULL;
+    if (object == NULL || object == Py_None || object == Py_False || (PyLong_CheckExact(object) && PyLong_AsLong(object) == 0)) {
+        *handle = NULL;
+        return true;
```

Comment:
we can't return true like this without python complaining I believe.

Reply:
this is the way to accept NONE, 0, False so Vspy releases the handle for the device. Python does not complain here